fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -203,11 +205,17 @@ private fun ReviewGradeCard(
     reviewGrade: ReviewQualityCheckState.Grade,
     modifier: Modifier = Modifier,
 ) {
-    ReviewQualityCheckCard(modifier = modifier.semantics(mergeDescendants = true) {}) {
+    ReviewQualityCheckCard(modifier) {
+        val title = stringResource(R.string.review_quality_check_grade_title)
+
         Text(
-            text = stringResource(R.string.review_quality_check_grade_title),
+            text = title,
             color = FirefoxTheme.colors.textPrimary,
             style = FirefoxTheme.typography.headline8,
+            modifier = Modifier.semantics {
+                heading()
+                contentDescription = "$title, Heading"
```

Comment:
Think you meant to reference the string resource here.

Reply:
Yes, thank you for catching that!